extension.js

```diff
@@ -71,7 +71,7 @@ async function insertSnippet(pos) {
 			pickedMethods.forEach(method => {
 				text += "\n\t" + method + "\n\t{\n\t\tthrow new \\Exception(\"Method not implemented\");\n\t}\n\t";
 			});
-			editor.insertSnippet(new vscode.SnippetString(text.replaceAll('$', '\\$')), pos);
+			editor.insertSnippet(new vscode.SnippetString(text.replace(/$/g, '\\$')), pos);
```

Comment:
So I figured out the problem. You forgot to escape the dollar sign because it's a control character in regular expressions